src/components/cqlLibraryLanding/CqlLibraryLanding.tsx

```diff
@@ -92,7 +92,10 @@ function CqlLibraryLanding() {
         }}
       />
       <div className="measure-table">
-        <section tw="flex flex-row">
+        <section
+          tw="flex flex-row"
+          style={{ borderBottom: "1px solid #b0b0b0" }}
```

Comment:
I think we need to handle it differently. there is already a line available if you see a couple of lines down after <Tabs> end. This is going to be a double line on it.
You will have to remove the the existing line or change its css. Also have to make sure that there is no gap between active tab indicator and this line.